app/Http/Controllers/Admin/PetShop/OwnerCrudController.php

```diff
@@ -62,7 +62,7 @@ protected function setupCreateOperation()
     {
         CRUD::setValidation(OwnerRequest::class);
 
-        CRUD::field('name');
+        CRUD::field('name')->wrapper(['class' => 'form-group col-md-6']);
```

Comment:
No need for this field to be smaller now that it doesn't have the google map. Also... `->size(6)` man... does the same thing... fluent syntax FTW! You're old-school, I know, I know 😅

```suggestion
        CRUD::field('name');
```